rules/integrations/aws/persistence_iam_user_created_access_keys_for_another_user.toml

```diff
@@ -0,0 +1,140 @@
+[metadata]
+creation_date = "2024/06/13"
+integration = ["aws"]
+maturity = "production"
+updated_date = "2024/06/13"
+min_stack_comments = "ES|QL rule type is still in technical preview as of 8.13, however this rule was tested successfully"
```

Comment:
At this point ES|QL is GA, do we want to change this?

Reply:
We can remove this min-stack comment or replace it with something like "ES|QL is available in 8.13 as technical preview."

Reply:
I'll change the min_stack comment to be more accurate